File: Foundation/lec02/prime.java
import java.util.*;
class prime{
    public static void main(String[] args){
        Scanner sr=new Scanner(System.in);
        System.out.println("enter number");
        int i = sr.nextInt();
      
    boolean isprime=true;
    for(int n=2; n<i; n++){
        if(i%n==0){
        isprime=false;
        break;
        }
    }
    if(isprime==true){
        System.out.print("prime");
        
    }
    else{
        System.out.println("not prime");

    
    }
}
}

File: Foundation/lec02/prime.cpp
#include<iostream>

using namespace std;
int main(){
    int i=0;
    cout<<"enter a number= ";
    cin>>i;
    bool isprime=true;
 for(int n=2; n<i; n++){
     if(i%n==0){
     isprime=false;
     break;
     }
 }
 if(isprime==true){
     cout<<"prime";
     
 }
 else{
cout<<"not prime";

 }
return 0;
}